machine/dni/dni_3048up/onie-rom.conf

```diff
@@ -0,0 +1,19 @@
+# DNI 3048UP onie ROM configuration
+
+description="Delta Networks, DNI-3048UP"
+
+# Top Down the Nand flash looks like:
+# 1. u-boot image  (include shmoo)
+# 2. u-boot environment
+# 3. onie-uimage
+format=contiguous-up
+
+uboot_machine=dni_34xx
+
+uimage_max_size=$(( 16 * 1024 * 1024 ))
+uboot_max_size=$((   3 * 1024 * 1024 ))
```

Comment:
Please confirm uimage_max_size and uboot_max_size are updated to reflect max sizes as per DTS.